tools/@aws-cdk/cfn2ts/lib/index.ts

```diff
@@ -42,6 +42,17 @@ export default async function(scopes: string | string[], outPath: string, option
 
 export async function generateAll(outPath: string, options: CodeGeneratorOptions) {
   const scopes = cfnSpec.namespaces();
+  const awsCdkLibDir = path.join(outPath, '..');
+
+  const pkgJsonPath = path.join(awsCdkLibDir, 'package.json');
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
+  const pkgJson = require(pkgJsonPath);
```

Comment:
NIT: Just for posterity because we use it more for these kinds of things as it's generally more predictable, I'd use `fs.readJson` (usually from fs-extra), instead of require for parsing json files.